locales/nl_NL.json

```diff
@@ -2306,23 +2306,23 @@
           "missing": "",
           "provided": "ID geüpload"
         },
-        "phone": {
-          "edit": "Edit phone number",
-          "label": "Phone number",
-          "add": "Add phone number",
-          "save": "Submit phone number",
-          "placeholder": "+1 000-123-1234"
+        "telefoon": {
```

Comment:
@borgqueenx Please don't translate the JSON key names (I'm guessing you're using machine translation?). This should still be called "phone".